Assets/LeapMotionModules/Interaction/InteractionBehaviourBase.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Leap;
+using Leap.Unity.Attributes;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.UI.Interaction {
+
+  public abstract class InteractionBehaviourBase : MonoBehaviour {
+
+    public InteractionManager interactionManager;
+
+    protected Rigidbody _body;
+    public new Rigidbody rigidbody { get { return _body; } }
+
+    [Header("Interaction Settings")]
+    public bool allowHover = true;
```

Comment:
Ah yeah I see, that makes sense hmmmmm.  That makes things tricky indeed, since like you say it is conceptually an override instead of an enabling.  I am fine with allow being the naming, and I agree that popping up a warning would be useful as well.